pkg/crc/machine/driver_darwin.go

```diff
@@ -26,3 +31,21 @@ func init() {
 
 	DefaultDriver = HyperkitDriver
 }
+
+func getDriverOptions(machineConfig config.MachineConfig) interface{} {
+	var driver interface{}
+
+	// Supported drivers
+	switch machineConfig.VMDriver {
+
+	case "virtualbox":
+		driver = virtualbox.CreateHost(machineConfig)
+	case "hyperkit":
+		driver = hyperkit.CreateHost(machineConfig)
+
+	default:
+		errors.ExitWithMessage(1, "Unsupported driver: %s", machineConfig.VMDriver)
+	}
+
+	return driver
+}
```

Comment:
New to code here so probably missing something but is this move of code really related to localising a string? If it's not, it should be a separate commit.